plugins/modules/win_feature_info.ps1

```diff
@@ -0,0 +1,48 @@
+#!powershell
+
+# Copyright: (c) 2020, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#AnsibleRequires -CSharpUtil Ansible.Basic
+#AnsibleRequires -CSharpUtil ansible_collections.ansible.windows.plugins.module_utils.SCManager
+
+$spec = @{
+    options             = @{
+        name = @{ type = "str"; default = '*'  }
+    }
+    supports_check_mode = $true
+}
+
+$module = [Ansible.Basic.AnsibleModule]::Create($args, $spec)
+
+$name = $module.Params.name
+
+$module.Result.exists = $false
+
+$features = Get-WindowsFeature -Name $name
+
+$module.Result.features = @(foreach ($feature in ($features)) {
+        # These should closely reflect the options for win_feature
+        [Ordered]@{
```

Comment:
Minor nit, there's not much point in using `[Ordered]` here. The keys are sorted alphabetically on the Ansible side and there is never a guarantee that the order will stay the same from PowerShell's perspective.

Reply:
completed